geometry_msgs/Pose2D[] waypoints
std_msgs/Float64[] cost
